src/agenda/reservation-list/style.js

```diff
@@ -12,6 +12,7 @@ export default function styleConstructor(theme = {}) {
     dayNum: {
       fontSize: 28,
       fontWeight: '200',
```

Comment:
Line 14 needs to be deleted

Reply:
You should submit the PR with all the changes you suggest and set the default to the existing value.